src/storage/QueryBaseProcessor.inl

```
@@ -375,15 +375,15 @@ kvstore::ResultCode QueryBaseProcessor<REQ, RESP>::collectEdgeProps(
                     return rank;
                 };
                 getters.getSrcTagProp = [&, this] (const std::string& tag,
-                                                   const std::string& prop) -> VariantType {
+                                                   const std::string& prop) -> OptVariantType {
                     auto it = fcontext->tagFilters_.find(std::make_pair(tag, prop));
                     CHECK(it != fcontext->tagFilters_.end());
```

Could we return an error to replace this ```CHECK```?